web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
This is not the place to discuss this, I will remove any further discussion on this topic as it's not helping with this PR. It also sounds like you're not understanding how React works correctly.

@florianorben please go with my earlier suggestion to use PostsView as the listener to the stores and propagate the changes through props to where they're needed.